src/Listener/JsonApiListener.php

```diff
@@ -67,7 +67,7 @@ class JsonApiListener extends ApiListener
      *
      * We attach at priority 10 so normal bound events can run before us
      *
-     * @return array
+     * @return mixed array or void
```

Comment:
nitpick: `void` shouldn't be mixed with any return type. Method should return `array` or `null`.